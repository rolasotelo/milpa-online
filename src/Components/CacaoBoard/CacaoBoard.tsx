import React from "react";

interface Props {}

const CacaoBoard = (props: Props) => {
  return <div className="h-16 bg-mexicanBoneLight">100 cacao</div>;
};

export default CacaoBoard;
