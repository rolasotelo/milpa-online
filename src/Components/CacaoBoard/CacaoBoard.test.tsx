import React from "react";
import renderer from "react-test-renderer";

import CacaoBoard from "./CacaoBoard";

describe("<CacaoBoard />", () => {
  const defaultProps = {};
  const wrapper = renderer.create(<CacaoBoard {...defaultProps} />);

  test("render", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
