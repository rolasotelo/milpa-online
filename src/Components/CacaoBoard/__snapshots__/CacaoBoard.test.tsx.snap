// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CacaoBoard /> render 1`] = `
<div
  className="h-16 bg-mexicanBoneLight"
>
  100 cacao
</div>
`;
